SalesforceCommerceCloud/pwa-kit
# [App Extensibility ⚙️] Update `Create App` to generate App Extension stubs (@W-16848439@)

# Description



The PR updates the generator to allow users to create an App Extension project with a typescript-minimal project for running and testing the created App Extension.

# Types of Changes




# Changes

- Updated generator to create an App Extension project.

# How to Test-Drive This PR

1. Run the generator locally 
```
node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js
```

2. Answer the questions:

```
? What type of project do you want to create? (Use arrow keys)
  PWA Kit Project
❯ Application Extension
? What is the name of your Application Extension?
The name must follow the pattern "@{namespace}/extension-{package-name}", where namespace is optional. @salesforce/extension-test
```

3. Start the generated App Extension by running `npm start` and verify the template typescript-minimal project starts and uses the App Extension. 

# Checklists




## General
